ts/Dialog/index.tsx

```diff
@@ -2,7 +2,7 @@ import {MDCDialog} from '@material/dialog/';
 import {bind} from 'bind-decorator';
 import {h} from 'preact';
 import MaterialComponent from '../Base/MaterialComponent';
-import Button from '../Button';
+import {Button} from '../Button';
```

Comment:
Shouldn't button be a default export?

Reply:
It has a default export but I don't know, if things like webpack can treeshake when you use the default export